webapp/app.ironcalc.com/frontend/src/components/WelcomeDialog/WelcomeDialog.tsx

```diff
@@ -0,0 +1,244 @@
+import { IronCalcIconWhite as IronCalcIcon } from "@ironcalc/workbook";
+import { Dialog, styled } from "@mui/material";
+import { House, Table, TicketsPlane, X } from "lucide-react";
+import { useState } from "react";
+import TemplatesListItem from "./TemplatesListItem";
+
+function WelcomeDialog(properties: {
```

Comment:
You call it `WelcomeDialog` but you are reusing it also as a `TemplateDialog` I think I would be more comfortable if you find a way so that these dialogs are semantically correct.